src/services/ui/src/api/useGetSubTypes.ts

```diff
@@ -0,0 +1,34 @@
+import { API } from "aws-amplify";
+import { useQuery, UseQueryOptions } from "@tanstack/react-query";
+import { opensearch, ReactQueryApiError } from "shared-types";
+import { subtypes } from "shared-types/opensearch";
+
+async function fetchSubTypes(
+  authorityId: number,
+  typeIds: number[]
+): Promise<opensearch.subtypes.Document[]> {
+  const endpoint = "/getSubTypes";
```

Comment:
can these be simplified since the functional pieces of code mimic that of `getTypes`? Reduces test writing needs and couples the idea of types/sub-types.